src/inc/quic_tls.h

```diff
@@ -230,6 +250,12 @@ typedef struct QUIC_TLS_PROCESS_STATE {
 
 } QUIC_TLS_PROCESS_STATE;
 
+//
+// Version of the wire-format for resumption tickets.
+// This needs to be incremented for each change in order or count of fields.
+//
+#define QUIC_TLS_RESUMPTION_TICKET_VERSION 1
```

Comment:
QUIC is the one doing the versioning, encoding and decoding, right? So this should be in the quicdef.h probably. Only TLS specific stuff goes in here.